projects/gitoxide/Dockerfile

```diff
@@ -16,6 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-rust
 RUN git clone --depth 1 https://github.com/Byron/gitoxide.git gitoxide
+RUN rustup update nightly
```

Comment:
I could imagine that it is preferable to update to well-known working version, rather than to the latest.
But then again, I think the dockerfile isn't executed each time, so the docker image itself constitutes a snapshot, which might be good enough.

Reply:
Yes, I think that there is some ambiguity in my phrase "by using the latest nightly toolchain": when fuzzing runs, I don't think this is guaranteed to be the latest at the time it runs.

As for what approach is best, I am really not sure. I did it this way because this is what was done in https://github.com/google/oss-fuzz/pull/12404/files and https://github.com/google/oss-fuzz/pull/12409/files, which seems to have worked for those projects.

Reply:
It's great there is a precedence, that should settle it.